app/routes.py

```diff
@@ -1,7 +1,11 @@
 from app import app
 from flask import render_template
+import sys
+sys.path.append('/scripts/pains/LillyMedchemRules/')
```

Comment:
Is this needed? Or did the __init__.py's fix the issue?

Reply:
you're right I forgot to remove it!